Source/WebCore/platform/graphics/mac/controls/WebControlView.mm

```diff
@@ -60,6 +60,8 @@ - (BOOL)isKeyWindow
     return _hasKeyAppearance;
 }
 
+// FIXME: These are hacks to prevent an AppKit crash when drawing an NSCell in GPUP
```

Comment:
Is there a radar you could leave here? Also, missing a period at the end of the first sentence.

Reply:
I modified the FIXME comment and I filed rdar://105250168.